Eth-Crowdfund/eth_crowdfund_be
# Campaigns by Manager

## What's this PR do?
* Adds the get_campaigns_by_manager_route and static method for Campaign
* Adds a test for happy and sad paths to the get_campaigns_by_manager route
## Where should the reviewer start? 
* `src/views/CampaignView.py`
* `src/tests/test_campaign.py`
## How should this be manually tested?
* Reviewer can run pytest and verify 8 passing tests
* Reviewer can fire up their server and hit `GET http://localhost:3000/api/v1/campaigns/manager/<manager>` with an existing manager address to view a happy path response and an incorrect manager address to view a sad path response.
## Any background context you want to provide?
* FE requested the ability to receive campaigns by manager address
## What are the relevant tickets?
